README.md

```diff
@@ -42,75 +52,123 @@ The repository is structured to support multi-team collaboration while maintaini
+### Alert structure
 
-At Giant Swarm we follow some best practices to organize our alerts:
+Giant Swarm supports two different types of alert rules:
 
-here is an example:
+#### 1. Metrics-based alerts
+
+These are standard Prometheus alerts based on PromQL queries against metrics data. They are stored in files ending with `.rules.yaml`.
+
+Example:
 
 ```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: management-cluster-app-failed-atlas
+  namespace: monitoring
+  labels:
+    area: platform
+    team: atlas
+spec:
   groups:
   - name: app
     rules:
     - alert: ManagementClusterAppFailedAtlas
-        annotations:
-            summary: Management cluster app not deployed correctly
-            description: '{{`Management Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
-            __dashboardUid__: UniqueID of the dashboard
-            __panelId__: id of the panel in the dashboard
-            dashboardQueryParams: "orgid=1"
-            # dashboardExternalUrl: https://link-to-my-dashboard
-            runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/app-failed/
-        expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",team="atlas"}
-        for: 30m
-        labels:
-            area: platform
-            cancel_if_cluster_status_creating: "true"
-            cancel_if_cluster_status_deleting: "true"
-            cancel_if_cluster_status_updating: "true"
-            cancel_if_outside_working_hours: "true"
-            severity: page
-            sig: none
-            team: atlas
+      annotations:
+        summary: Management cluster app not deployed correctly
+        description: '{{`Management Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
+        __dashboardUid__: unique-id-of-the-dashboard
+        __panelId__: id of the panel in the dashboard
+        dashboardQueryParams: "orgid=1"
+        # dashboardExternalUrl: https://link-to-my-dashboard
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/app-failed/
+      expr: app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog=~"control-plane-.*",team="atlas"}
+      for: 30m
+      labels:
+        area: platform
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        sig: none
+        team: atlas
+```
+
+#### 2. Logs-based alerts
+
+These alerts are generated from log data using LogQL queries processed by Loki. To create a log-based alert:
+
+- Add the label `application.giantswarm.io/prometheus-rule-kind: loki` to the alert
```

Comment:
Also why is there both a label and a naming convention for LogQL based alerts ?

Reply:
I think it will be easier to see them in a folder and also we need some differentiator when we add the testing tooling

Reply:
Ok so naming is for the repository organization and the label for deployment purposes.

Reply:
But maybe helm could automatically add the label for files with this extension?

Reply:
You're right, I edited the _helpers.tpl to add the alert label if the files ends with `logs.yml` or `logs.yaml`